clu/src/lib.rs

```diff
@@ -137,17 +135,21 @@ pub fn cleanup() -> Result<(), Error> {
 
     for i in interfaces {
         if re.is_match(&i) {
-            KI.del_interface(&i)?;
+            KI.del_interface(&i).expect("failed to delete a w# interface");
```

Comment:
Are we not able to at least tell them which interface that was?

Reply:
Also, intuition tells me cleanup routines wouldn't usually stop after a problem appears, i.e. if someone manually deletes an interface which is on this list and `del_interface` fails because of that, why should that ever affect the rest of the cleanup?

Reply:
this was just an oversight on my part I'll put it back to ?